rasa_core/restore.py

```diff
@@ -57,13 +58,49 @@ def _check_prediction_aligns_with_story(last_prediction,
     # type: (List[Text], List[Text]) -> None
     """Emit a warning if predictions do not align with expected actions."""
 
-    p, a = evaluate.align_lists(last_prediction, actions_between_utterances)
+    p, a = align_lists(last_prediction, actions_between_utterances)
     if p != a:
         warnings.warn("Model predicted different actions than the "
                       "model used to create the story! Expected: "
                       "{} but got {}.".format(p, a))
 
 
+def align_lists(pred, actual):
```

Comment:
let's name it `predictions`, `golds` here, or `preds`, `actuals`, but then rename `gold` to `actual` everywhere